formative/2022-10-20.md

```diff
@@ -0,0 +1,29 @@
+# Atbildes uz jautājumiem
+
+1. Klase ir objekta paraugs - vispārināta versija.
+2. Objekts ir kāds priekšmets programmā, kas veidots pēc klases parauga un, kuram atšķirībā no klases ir unikāls nosaukums un īpašības ar vērtībām.
+3. Objekta īpašības ir mainīgie, kas raksturo objektu un ar kuriem ir iespējams darboties ar objektu.
+4. Metode ir funkcija, kas atrodas klasē.
+5. Konstruktors ir metode, kas ļauj piešķirt objektam īpašības, kad tas tiek izveidots. To izmanto, lai nebūtu tik gari jāraksta, kad piešķir objektam īpašības.
+
+## Klases piemērs
+```php
+class Bottle
+{
+    public string $material;
+    public float $height;
+    public float $diameter;
+    public float $volume;
+    
+    function __construct(string $material, float $height, float $diameter)
+    {
+        $this->material = $material;
+        $this->height = $height;
+        $this->diameter = $diameter;
+        $volume = 3.14*(0.5*$diameter)*(0.5*$diameter)*height;

```

Comment:
Šis $volume mainīgais nav uzstādīta kā klases īpašība, līdz ar to, ārpus __constructor tā nebūs pieejama